python-lib/osisoft_client.py

```diff
@@ -370,7 +370,7 @@ def get_debug_level(self):
 
     def get_resource_path_params(self, resource_path):
         return {
-            "path": escape(resource_path)
+            "path": resource_path
```

Comment:
We should keep the escape() in and fix those double encoding issues with the function

Reply:
@liamlynch-data the escape is now fixed and reinstated in [d1523b4](https://github.com/dataiku/dss-plugin-osisoft/pull/35/commits/d1523b47e4f85371606de602081612847015a7a7) and [f064232](https://github.com/dataiku/dss-plugin-osisoft/pull/35/commits/f06423287d8657cf8ca2995ec3bc4363fbebe7be) of the last [PR](https://github.com/dataiku/dss-plugin-osisoft/pull/35).

Reply:
Yep all good - tested and approved in the other PR